distribution/test/itests/pom.xml

```diff
@@ -125,6 +117,61 @@
             <artifactId>javax.inject</artifactId>
         </dependency>
         <!-- End of pax exam dependencies -->
+        <dependency>
```

Comment:
Most of them are. A few could be pushed down into test-itests-ddf. I'll see what I can do.